src/main/java/uk/gov/hmcts/darts/event/model/DarNotifyApplicationEvent.java

```diff
@@ -8,10 +8,12 @@
 public class DarNotifyApplicationEvent extends ApplicationEvent {
     private final DartsEvent dartsEvent;
     private final DarNotifyType darNotifyType;
+    private final Integer courtroomId;
```

Comment:
```suggestion
    private final Integer courtRoomId;
```

Reply:
done, fixed in developer so that it could change all usages